layouts/page/pricing.html

```diff
@@ -1,55 +1,114 @@
 {{ define "hero" }}
     <header class="header-hero">
         <div class="container mx-auto text-center flex-col max-w-2xl">
-            <h1>Plans for teams of all sizes.</h1>
-            <p>
-                Enable your developers and operators to create, deploy, and manage
-                applications and infrastructure on AWS, Azure, Google Cloud, and
-                Kubernetes. Whether you are a growing startup, or established enterprise,
-                Pulumi has a solution for you.
-            </p>
-            <p>
-                <a class="rounded text-orange-400 font-bold underline md:font-normal md:no-underline md:border-2 md:border-orange-400 md:py-1 md:px-2 md:mr-1 hover:text-orange-300 hover:border-orange-300 transition-all"
-                    href="#community-edition">Pulumi Community Edition</a>
-                is free forever for unlimited individual use.
-            </p>
+            <h1>Pulumi for every team and individual.</h1>
+            <div class="container mx-auto lg:flex justify-center mt-8">
+                <div class="px-8 lg:w-1/2 text-center flex flex-col text-gray-300">
+                    <div class="text-lg uppercase rounded text-blue-300 font-bold">Open source</div>
+                    <p>
+                        Pulumi is an
+                        <a href="https://github.com/pulumi/pulumi" class="text-orange-500 hover:text-orange-400 font-bold">
+                            open source project</a>.
+                        The <a href="#community-edition" class="text-orange-500 hover:text-orange-400 font-bold">
+                            Community Edition</a> is available for free,
+                        or you can
+                        <a href="{{ relref . "/docs/intro/concepts/state#self-managed-backend" }}"
+                            class="text-orange-500 hover:text-orange-400 font-bold">
+                            run Pulumi offline</a> and manage state yourself.
+                    </p>
+                </div>
+                <div class="px-8 lg:w-1/2 text-center flex flex-col">
+                    <div class="text-lg uppercase rounded text-blue-300 font-bold">Great for teams.</div>
+                    <p>
+                        <a href="#team-edition" class="text-orange-500 hover:text-orange-400 font-bold">
+                            Team Starter and Pro</a> help teams collaborate on shared projects.
+                        <a href="#enterprise-edition" class="text-orange-500 hover:text-orange-400 font-bold">
+                            Pulumi Enterprise</a> adds security controls and self-host options.
+                    </p>
+                </div>
+            </div>
         </div>
     </header>
 {{ end }}
 
 {{ define "main" }}
-    <section class="bg-purple-800 pt-24 text-white px-4 lg:px-0">
+    <section class="bg-purple-800 pt-24 text-white px-4 lg:px-0 justify-center text-center">
+        <h2 class="text-gray-300 mb-8">Pulumi Editions</h2>
```

Comment:
You could win back a little vertical space by dropping this, since it's not really necessary.

Reply:
Sounds great. Done!